Sources/PixelKit/PixelKit.swift

```diff
@@ -233,7 +240,7 @@ public final class PixelKit {
             } else {
                 printDebugInfo(pixelName: pixelName + "_d", frequency: frequency, parameters: newParams, skipped: true)
             }
-        case .dailyAndCount:
+        case .legacyDailyAndCount:
```

Comment:
This change could trip up anyone who already has a new pixel on a branch that uses `dailyAndCount`, as the meaning of that value will change after this PR. I'll announce this change to the team before merging it to mitigate this.